packages/input-container/test/visual/lumo/input-container.test.js

```diff
@@ -35,6 +35,12 @@ describe('input-container', () => {
     await visualDiff(div, 'invalid');
   });
 
+  it('readonly invalid', async () => {
+    element.readonly = true;
+    element.invalid = true;
```

Comment:
Thanks for the fix 👍 would it make sense to add text in the field to ensure that it's readable?

Reply:
Yes, probably. We don't have visual tests for regular `readonly` with `value`. Created an issue: #7207